src/shared_utilities/FileReader.jl

```diff
@@ -205,8 +216,18 @@ function PrescribedDataTemporal{FT}(
             outfile_root;
             mono = mono,
         )
+
+        NCDataset(infile_path, "r") do ds
+            if !("time" in keys(ds))
+                error(
+                    "Using a temporal albedo map requires data with time dimension.",
+                )
+            end
+        end
     end
     ClimaComms.barrier(comms_ctx)
+    infile_path = get_infile_path()
```

Comment:
similar question here

Reply:
hopefully the comment above explains both!

Reply:
yep! this makes sense to me, I see the `barrier` here 👍